src/meta/src/stream/graph/stream_graph.rs

```diff
@@ -475,6 +484,24 @@ where
             ),
             _ => {
                 let mut new_stream_node = stream_node.clone();
+                if let Node::HashJoinNode(node) = new_stream_node
+                    .node
+                    .as_mut()
+                    .ok_or(ProstFieldNotFound("prost stream node field not found"))?
```

Comment:
A little stupid here. I found I can not convert a Result<&mut T, &mut Err> to Result<&mut T, Err> therefore I just create a new error.. Asking rustaceans

Reply:
Just unwrap here? The frontend should ensure this field exists.

Reply:
Missed these.... Will try to fix your comments in next PRs.